src/packages/DAO/elections.dao.ts

```diff
@@ -38,31 +44,55 @@ export async function createElection(formData: CreateElection) {
 export async function startElection(formData: FormData) {
   const electionId = String(formData.get("electionId"));
   const updateElectionSchema = z.string().uuid();
-  const parsedElectionId = updateElectionSchema.parse(electionId);
+  const parsedElectionId = updateElectionSchema.safeParse(electionId);
+
+  if (!parsedElectionId.success) {
+    throw new BadRequestError(parsedElectionId.error.name);
+  }
 
   await prisma.election.update({
-    where: { id: parsedElectionId },
+    where: { id: parsedElectionId.data },
     data: {
       name: "Updated on server",
       status: ElectionStatus.ONGOING,
     },
   });
 
-  revalidatePath("/");
+  redirect(`http://localhost:3000/elections/${electionId}`);
```

Comment:
afoid hard-coding URLs, as they can change for each environment (production, staging, local, so on)

Is this case, you can just remove it,

```suggestion
  redirect(`http://localhost:3000/elections/${electionId}`);
```

but if you really need the origin URL, use a env var to getting

```ts
  // with env var
  redirect(`/elections/${electionId}`);
```